lib/internal/modules/cjs/loader.js

```diff
@@ -766,6 +767,17 @@ Module._load = function(request, parent, isMain) {
   }
 
   const filename = Module._resolveFilename(request, parent, isMain);
+  if (StringPrototypeStartsWith(filename, 'node:')) {
+    // Slice 'node:' prefix
+    const id = StringPrototypeSlice(filename, 5);
+
+    const module = loadNativeModule(id, request);
+    if (!module?.canBeRequiredByUsers) {
+      throw new ERR_UNKNOWN_BUILTIN_MODULE(filename);
+    }
+
+    return module.exports;
+  }
```

Comment:
What is the motivation for short-circuiting the cache here, something that has never been done previously for native modules?

I don't see why this scheme should be any different to any other in this regard.

Reply:
On this topic, should we throw or defer to cache on unknown builtins?